src/stories/fixtures.ts

```diff
@@ -0,0 +1,172 @@
+  pr_url: 'https://github.com/test/test-repo/pull/1234',
+  pr_is_open: true,
+  pr_is_merged: false,
+  has_unmerged_commits: true,
+  currently_creating_pr: false,
+  currently_fetching_org_config_names: false,
+  github_users: [sampleGitHubUser1, sampleGitHubUser2, sampleGitHubUser3],
+  status: EPIC_STATUSES.IN_PROGRESS,
+  available_task_org_config_names: [],
+};
+
+export const sampleEpic3 = {
+  id: 'e3',
+  project: 'p1',
+  name: 'Regular Database Backups',
+  slug: 'database-backups',
+  old_slugs: [],
+  description: 'Fulfilling the requirements to access specific tech specs.',
+  description_rendered:
+    '<p>Fulfilling the requirements to access specific tech specs.</p>',
+  branch_name: 'feature/database-backups',
+  branch_url: 'https://github.com/test/test-repo/tree/feature/database-backups',
+  branch_diff_url:
+    'https://github.com/test/test-repo/compare/main...feature/database-backups',
+  pr_url: null,
+  pr_is_open: false,
+  pr_is_merged: false,
+  has_unmerged_commits: false,
+  currently_creating_pr: false,
+  currently_fetching_org_config_names: false,
+  github_users: [],
+  status: EPIC_STATUSES.PLANNED,
+  available_task_org_config_names: [],
+};
+
+export const sampleEpic4 = {
+  id: 'e4',
+  project: 'p1',
+  name: 'Data Controls',
+  slug: 'data-controls',
+  old_slugs: [],
+  description: 'Links the controller field to the static items',
+  description_rendered: '<p>Links the controller field to the static items</p>',
+  branch_name: 'feature/data-controls',
+  branch_url: 'https://github.com/test/test-repo/tree/feature/data-controls',
+  branch_diff_url:
+    'https://github.com/test/test-repo/compare/main...feature/data-controls',
+  pr_url: null,
+  pr_is_open: false,
+  pr_is_merged: false,
+  has_unmerged_commits: false,
+  currently_creating_pr: false,
+  currently_fetching_org_config_names: false,
+  github_users: [],
+  status: EPIC_STATUSES.MERGED,
+  available_task_org_config_names: [],
+};
+
+export const sampleTask1 = {
+  id: 't1',
+  name: 'Data Mapping',
+  slug: 'data-mapping',
+  old_slugs: [],
+  epic: 'e1',
+  description: 'This is a description',
+  description_rendered: '<p>This is <em>safely</em> rendered Markdown.</p>',
+  has_unmerged_commits: true,
+  currently_creating_pr: false,
+  branch_name: 'feature/my-epic__data-mapping',
+  branch_url:
+    'https://github.com/test/test-repo/tree/feature/my-epic__data-mapping',
+  branch_diff_url:
+    'https://github.com/test/test-repo/compare/feature/my-epic...feature/my-epic__data-mapping',
+  pr_url: 'https://github.com/test/test-repo/pull/1357',
+  pr_is_open: true,
+  commits: [sampleCommit1],
+  origin_sha: '723b342',
+  assigned_dev: sampleGitHubUser1,
+  assigned_qa: null,
+  status: TASK_STATUSES.IN_PROGRESS,
+  currently_submitting_review: false,
+  review_submitted_at: '2019-03-01T19:47:49Z',
+  review_valid: true,
+  review_status: REVIEW_STATUSES.APPROVED,
+  review_sha: '617a512',
+  org_config_name: 'dev',
+};
```

Comment:
@duggiemitchell @stacyk @dvdherron If you three prefer to split this up into separate files (e.g. `/fixtures/epics.ts`, `/fixtures/tasks.ts` etc) that's fine with me -- your call.